### PR TITLE
AI-150:Fixed modal buttons regression issue

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
Main part of this issues is fixed at https://github.com/compucorp/org.civicrm.shoreditch/pull/28

This PR fixes 
- margin for .fa-icon 
- hover/active effect of button

**Before:**
before-4

**After:**
after-4

**Note:**
This PR resolves button issue in popup throughout CiviCRM, backstopjs tests passed.

### CODE REVIEW

**Comment by AkA84** [comment]
@mukeshcompucorp if this is a regression issue, please explain where the regression happen in the PR description

**Comment by AkA84** [comment]
@mukeshcompucorp also please check this PR https://github.com/compucorp/org.civicrm.shoreditch/pull/28 , because it seems to me that the issue is the same

File: scss/civicrm/common/_modals.scss [hunk 1]
```diff
@@ -141,15 +141,23 @@
           margin-left: 0 !important;
         }
 
-        &:hover {
+        &:hover,
+        &:active,
+        &:focus {
           color: $crm-white;
-          opacity: .8;
+          opacity: .8 !important;
```
Comment (AkA84) [inline scss/civicrm/common/_modals.scss]: why the !important?

Comment (mukeshcompucorp) [reply scss/civicrm/common/_modals.scss]: the button extends the %btn-civi which has opacity: 1 !important, to override that we need important here.

**Pull request merged**